build.gradle

```diff
@@ -61,14 +61,17 @@ configure(subprojects.findAll {it.name.contains("hotfix")}) {
     }
 
     dependencies {
-        alfrescoProvided("${alfRepoDep}")
-        alfrescoProvided("${quartz}")
+        alfrescoProvided "${alfRepoDep}"
+        alfrescoProvided "${quartz}"
         baseAlfrescoWar "${alfrescoWarDep}"
         alfrescoAmp project.tasks.amp.outputs.files
     }
 
     repositories {
         mavenCentral()
+        maven {
+            url 'https://artifacts.alfresco.com/nexus/content/repositories/public/'
```

Comment:
The `eu.xenit.alfresco` plugin provides an `alfrescoPublic()` shortcut method to set this maven repository.